test_docker.sh

```diff
@@ -1 +0,0 @@
-docker run --rm -v $(pwd):/Qt.py mottosso/qt.py27
```

Comment:
Ok, I added all three – which I also think is how it's supposed to be done, now that some parts of tests can not be performed by all Python versions.